smokey/src/main/java/frc/robot/swerveHelpers/Falcon500SteerControllerFactoryBuilder.java

```diff
@@ -201,10 +201,10 @@ public void setReferenceAngle(double referenceAngleRadians) {
                 if (++resetIteration >= ENCODER_RESET_ITERATIONS) {
```

Comment:
If the robot is enabled before this happens the first time, the robot is disoriented.  
Is there a way to ensure this happens without delay at construction?
If not, should we set a flag on the first time through, and then print the message once to alert the user it's OK to enable.